components/error_page/error_page.tsx

```diff
@@ -77,7 +81,7 @@ export default class ErrorPage extends React.PureComponent {
             );
         } else if (type === ErrorPageTypes.CHANNEL_NOT_FOUND && isGuest) {
             backButton = (
-                <Link to={params.get('returnTo')}>
+                <Link to={params.get('returnTo') as string}>
```

Comment:
The `as string` here is to prevent errors - the `URLSearchParams` type provides it as `string | null` and `to` in `<Link/>` does not accept `null`.